src/main/kotlin/no/nav/familie/pdf/pdf/PdfController.kt

```diff
@@ -1,14 +1,18 @@
 package no.nav.familie.pdf.pdf
 
-import no.nav.security.token.support.core.api.Unprotected
+import no.nav.familie.sikkerhet.EksternBrukerUtils
+import no.nav.security.token.support.core.api.ProtectedWithClaims
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@Unprotected
 @RequestMapping("api/pdf")
+@ProtectedWithClaims(
+    issuer = EksternBrukerUtils.ISSUER_TOKENX,
+    claimMap = ["acr=Level4"],
+)
```

Comment:
```suggestion
@ProtectedWithClaims(
   issuer = "azuread"
)
```

Dette er ikke mitt spesialfelt men lurer på om du må sjekke på Azure AD id som utsedet for interne maskin-maskin kall slik som dette. 

TokenX er ment for person-interne-maskin kall